webapplication/user/views.py

```diff
@@ -24,6 +29,21 @@ def perform_create(self, serializer):
         return user
 
 
+class CustomUserDetailsView(UserDetailsView):
+    def get(self, request, *args, **kwargs):
+        response = super().get(request, *args, **kwargs)
+        component_with_remote_executions = Component.objects.filter(Q(success=True) & Q(geoap_creds_required=True))
+        response.data['server_for_calculation_is_needed'] = component_with_remote_executions.exists()
+        if component_with_remote_executions.exists():
+            server_is_overloaded_notification = switch_is_active('our_server_is_overloaded_notification')
+            remote_server_available = switch_is_active('remote_server_available')
+            if server_is_overloaded_notification:
+                response.data['remote_server_available'] = False
+            else:
+                response.data['remote_server_available'] = remote_server_available
```

Comment:
Here you'll need only one field - show or not the notification on FE

Reply:
our_server_is_overloaded_notification we need for force notification